service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/PractitionerRoleMapper.java

```diff
@@ -47,8 +51,29 @@
         return roles;
     }
 
-    private CodeableConcept getCode(POCDMT000002UK01AssignedAuthor author) {
-        CE code = author.getCode();
+    public PractitionerRole mapResponsibleParty(POCDMT000002UK01ClinicalDocument1 clinicalDocument) {
+        PractitionerRole role = new PractitionerRole();
```

Comment:
So we need a role even if `clinicalDocument.isSetComponentOf() == false` and it will be empty?

Reply:
fixed